src/QMCWaveFunctions/SPOSet.h

```diff
@@ -218,9 +219,7 @@ class SPOSet: public QMCTraits
   virtual void
   evaluate (const ParticleSet& P, PosType &r, ValueVector_t &psi)
   {
-    app_error() << "Need specialization for SPOSet::evaluate "
-                << "(const ParticleSet& P, PosType &r).\n";
-    abort();
+    APP_ABORT("Need specialization for SPOSet::evaluate(const ParticleSet& P, PosType &r)\n");
```

Comment:
Aiming at reducing header inclusion. Please move the function definition into SPOSet.cpp and remove "#include "Message/Communicate.h"

Reply:
Good point. Updated.